srml/support/procedural/src/storage/transformation.rs

```diff
@@ -181,12 +181,14 @@ fn decl_store_extra_genesis(
 
 		let mut opt_build;
 		// need build line
-		if let (Some(ref getter), Some(ref config)) = (getter, config) {
+		if let Some(ref config) = config {
 			let ident = if let Some(ident) = config.expr.content.as_ref() {
 				quote!( #ident )
-			} else {
+			} else if let Some(ref getter) = getter {
 				let ident = &getter.getfn.content;
 				quote!( #ident )
+			} else {
```

Comment:
Seems that the ident from config is preferred over the one in getter. Should we make sure that if both are defined they are the same?

Reply:
Oh is this wanted ? maybe I should do test but I think it should work with different ident.

Reply:
I mean I don't have that much opinion on it but right now it should work with different ones